src/components/Chart.vue

```diff
@@ -10,17 +10,14 @@ export default {
     props: ['options'], 
     mounted () {
         this.renderChart(this.chartData, this.options)
-    },
-    updated () {
-        console.log('⚡ I am updated');
-    },
-    methods: {
-        update(newData) {
-            console.log('updating', newData)
-            this.chartData.datasets.push(newData)
-            this._chart.update()      
-        }
     }
+    // methods: {
```

Comment:
comments are not good :) if you are not sure of your code, you should delete this :) GIT repository has a history of your changes.